src/styles/Container.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+export const Container = styled.section`
```

Comment:
i think this should just be a div?

Reply:
i chose section over div for semantic purposes, and perhaps reuse it for dashboard's sections as well